test/functions/operatorOverloads/operatorMethods/allOps/ioOperator.good

```diff
@@ -1 +1,5 @@
+ioOperator.chpl:7: warning: the <~> operator is deprecated
+ioOperator.chpl:13: In function 'main':
+ioOperator.chpl:15: warning: the <~> operator is deprecated
+ioOperator.chpl:15: warning: the <~> operator is deprecated
```

Comment:
I think it'd be fair to remove the IO operator from this test instead

Reply:
Offline we discussed this a bit, and I'm opting to keep these tests around as-is and remove them post-release.